### PR TITLE
Chelsbio

### PR DESCRIPTION
**Pull request by camckin10**
screen shot 2018-06-14 at 12 54 50 pm

This is what my bio looks like on the website in the Community section.

### CODE REVIEW

**Comment by camckin10** [comment]
A Note:
*Named the js file that holds my bio my first and last name. (ChelseyMcKinney.js)

File: src/components/Contact.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
 import React from 'react';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+//import  from '@fortawesome/fontawesome-free-brands/faCamera';
```
Comment (experimatt) [inline src/components/Contact.js]: Remove these if we're not using them

File: src/components/Contact.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+import faSlack from '@fortawesome/fontawesome-free-brands/faSlack';
+import faTwitter from '@fortawesome/fontawesome-free-brands/faTwitter';
+//import email from '@fortawesome/fontawesome-free-solid/faEnvelope';
```
Comment (experimatt) [inline src/components/Contact.js]: Remove if we're not using

**Comment by experimatt** [comment]
It looks like you based this new branch off of your `chelscontact` branch, and so when compared to master it looks like you're changing things in the `Contact.js` component.

You may need to merge master into this branch once we merge `chelscontact` into `master.

**Comment by camckin10** [comment]
Got it. For now I will work on the font awesome icons..

**Comment by experimatt** [comment]
@camckin10 this looks good except for the commented out font awesome imports. Once you remove them I'll happily review and approve!

**Pull request merged**